www/templates/layouts/default.php

```diff
@@ -6,7 +6,10 @@
 global $supportsAuth;
 global $supportsSaml;
 global $COMPACT_MODE;
+
+require_once __DIR__ . '/../../common.inc';
 ?>
+  <title><?php $page_title ? $page_title : 'WebPageTest' ?></title>
```

Comment:
Think we're missing an echo here @jefflembeck ? 

```
<title><?php echo $page_title ? $page_title : 'WebPageTest' ?></title>
```

Reply:
Whew yes good call.